printer_mods/baccula/revo_skirt_nozzle_holder/README.md

```diff
@@ -0,0 +1,24 @@
+# Revo Nozzle Holder v2.4 350mm
+Remix of [Andre's](https://github.com/VoronDesign/VoronUsers/tree/master/printer_mods/Andre/Revo_Nozzle_Skirt_Trident) skirt-mounted nozzle holder to fit Voron 2.4 350mm printers, the BOM and installation steps are the same.
+
+This will fit up to 4 nozzles, and uses the '300mm' part from Andre's design, unmodified.
```

Comment:
you mention 300mm parts here, but the first sentence says it's designed to fit the Voron 2.4 350mm , by the filename, 350mm appears to be correct, so, is this sentence wrong?

Reply:
Due to the differences in skirt design, the 300mm Trident skirt is used, even though this mod only fits the 350mm v2. I added a clarifying sentence to the README.